src/examples/starter-apps/chatbot.py

```diff
@@ -0,0 +1,161 @@
+import asyncio
+import aiohttp
+import logging
+import os
+from PIL import Image
+from typing import AsyncGenerator
+
+from dailyai.pipeline.aggregators import (
+    LLMAssistantContextAggregator,
+    LLMResponseAggregator,
+    LLMUserContextAggregator,
+    UserResponseAggregator,
+)
+from dailyai.pipeline.frames import (
+    ImageFrame,
+    SpriteFrame,
+    Frame,
+    LLMResponseEndFrame,
+    LLMResponseStartFrame,
+    LLMMessagesQueueFrame,
+    UserStartedSpeakingFrame,
+    AudioFrame,
+)
+from dailyai.services.ai_services import AIService
+from dailyai.pipeline.pipeline import Pipeline
+from dailyai.services.ai_services import FrameLogger
+from dailyai.services.daily_transport_service import DailyTransportService
+from dailyai.services.open_ai_services import OpenAILLMService
+from dailyai.services.elevenlabs_ai_service import ElevenLabsTTSService
+from examples.support.runner import configure
+
+logging.basicConfig(format=f"%(levelno)s %(asctime)s %(message)s")
+logger = logging.getLogger("dailyai")
+logger.setLevel(logging.DEBUG)
+
+sprites = []
+
+script_dir = os.path.dirname(__file__)
+
+for i in range(1, 26):
+    # Build the full path to the image file
+    full_path = os.path.join(script_dir, f"assets/robot0{i}.png")
+    # Get the filename without the extension to use as the dictionary key
+    # Open the image and convert it to bytes
+    with Image.open(full_path) as img:
+        sprites.append(img.tobytes())
+
+flipped = sprites[::-1]
+sprites.extend(flipped)
+# When the bot isn't talking, show a static image of the cat listening
+quiet_frame = ImageFrame("", sprites[0])
+talking_frame = SpriteFrame(images=sprites)
+
+
+class TalkingAnimation(AIService):
```

Comment:
Should this have FrameProcessor as a parent?

Reply:
This one is still using `run_to_queue`, so it doesn't like being a `FrameProcessor` :(